### PR TITLE
Add org.freedesktop.Sdk.Extension.php74

### PR DESCRIPTION
**Pull request by Lctrs**
To be merged in branch `branch/19.08`.

@p1u3o and @stobbsm Do you want to be added as maintainers as well ?

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build org.freedesktop.Sdk.Extension.php74

**Comment by p1u3o** [comment]
@Lctrs Please :+1:

**Comment by Lctrs** [comment]
@LeandroStanger Okay it should build now.

**Comment by LeandroStanger** [comment]
bot, build org.freedesktop.Sdk.Extension.php74

File: org.freedesktop.Sdk.Extension.php74.json [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+				{
+					"type": "script",
+					"dest-filename": "enable.sh",
+					"commands": [
+						"export PATH=$PATH:/usr/lib/sdk/php74/bin:/var/config/composer/vendor/bin"
+					]
+				},
+				{
+					"type": "file",
+					"path": "org.freedesktop.Sdk.Extension.php74.appdata.xml"
```
Comment (LeandroStanger) [inline org.freedesktop.Sdk.Extension.php74.json]: New appdata files should use metainfo instead. https://freedesktop.org/software/appstream/docs/chap-Quickstart.html

Comment (Lctrs) [reply org.freedesktop.Sdk.Extension.php74.json]: Done.

**Comment by LeandroStanger** [comment]
bot, build org.freedesktop.Sdk.Extension.php74

**Comment by stobbsm** [comment]
I'm willing to be a maintainer. Just dealing with some health issues keeping me away from fun stuff for a bit.

**Comment by bilelmoussaoui** [comment]
@stobbsm Invitation sent.

**Pull request closed without merging**